src/Serializer.php

```diff
@@ -562,7 +508,7 @@ private static function getPhpArraySerializer()
 
     public static function loadKnownMetadataTypes()
     {
-        foreach (self::$metadataKnownTypes as $key => $class) {
+        foreach (KnownTypes::allKnownTypes() as $key => $class) {
```

Comment:
nit: Why not just 
```php
        foreach (KnownTypes::GRPC_TYPES as $key => $class) {
```

The values are the same for both GRPC and JSON types. Potentially you could also do this
```php
        foreach (KnownTypes::GRPC_TYPES + KnownTypes::JSON_TYPES as $key => $class) {
```